### PR TITLE
D3D11 video decoder supports more codecs.

### PR DESCRIPTION
**Pull request by jianjunz**

### CODE REVIEW

File: scripts/build-win.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ def gngen(arch, sio_root, ffmpeg_root, ssl_root, msdk_root, quic_root, scheme, t
         gn_args.append('owt_ffmpeg_root="%s"'%(ffmpeg_root))
     if ffmpeg_root or msdk_root or cg_server:
         gn_args.append('rtc_use_h264=true')
-    if msdk_root or cg_server:
+    if msdk_root or cg_server or cg_client:
```
Comment (taste1981) [inline scripts/build-win.py:100]: we intend to enable this also for ffmpeg_root?

Comment (jianjunz) [reply scripts/build-win.py:100]: We need to align the behavior with H.264. Is there any license issue to enable it?

**Review by taste1981 — APPROVED** [review]
+1 with nits

File: scripts/build-win.py (hunk 1 above)
Comment (taste1981) [reply scripts/build-win.py:100]: There should be no. We don't compile ffmpeg during our build process, but only provide instruction on how uses integrate their own ffmpeg library.

Comment (jianjunz) [reply scripts/build-win.py]: Added `ffmpeg_root`, removed `cg_client`. Same as H.264.

**Review by taste1981 — APPROVED** [review]

**Pull request merged**